Pinta.Effects/Effects/RadialBlurEffect.cs

```diff
@@ -120,43 +116,21 @@ private static ColorBgra GetFinalPixelColor (
 				X: o1.X + settings.fcx + 32768 >> 16,
 				Y: o1.Y + settings.fcy + 32768 >> 16);
 
-			if (p1.X > 0 && p1.Y > 0 && p1.X < settings.canvasSize.Width && p1.Y < settings.canvasSize.Height) {
-
-				ColorBgra sample = sourceData[p1.Y * settings.canvasSize.Width + p1.X];
-
-				sr += sample.R;
-				sg += sample.G;
-				sb += sample.B;
-				sa += sample.A;
-
-				++sc;
-			}
+			if (p1.X > 0 && p1.Y > 0 && p1.X < settings.canvasSize.Width && p1.Y < settings.canvasSize.Height)
+				blender += sourceData[p1.Y * settings.canvasSize.Width + p1.X];
 
 			PointI p2 = new (
 				X: o2.X + settings.fcx + 32768 >> 16,
 				Y: o2.Y + settings.fcy + 32768 >> 16);
 
-			if (p2.X > 0 && p2.Y > 0 && p2.X < settings.canvasSize.Width && p2.Y < settings.canvasSize.Height) {
-
-				ColorBgra sample = sourceData[p2.Y * settings.canvasSize.Width + p2.X];
-
-				sr += sample.R;
-				sg += sample.G;
-				sb += sample.B;
-				sa += sample.A;
-
-				++sc;
-			}
+			if (p2.X > 0 && p2.Y > 0 && p2.X < settings.canvasSize.Width && p2.Y < settings.canvasSize.Height)
+				blender += sourceData[p2.Y * settings.canvasSize.Width + p2.X];
 		}
 
-		return
-			(sa > 0)
-			? ColorBgra.FromBgra (
-				b: Utility.ClampToByte (sb / sc),
-				g: Utility.ClampToByte (sg / sc),
-				r: Utility.ClampToByte (sr / sc),
-				a: Utility.ClampToByte (sa / sc))
-			: ColorBgra.Transparent;
+		if (blender.Aggregate.A > 0)
```

Comment:
I'm not sure why the `sa > 0` check was in the original code .. I would have expected it to check `sc > 0` instead because of the division. But `sc` should be greater than 0 since we always have at least the one initial sample?
If the total alpha is zero the result is going to be a transparent color anyways

Reply:
Ah, there was a division by `sa` in the original code, which changed in 7e4bf3dd45e233b060e5ad4e6a12860ca2dc93cb
So I think this could just directly call `Blend()` now